model/load_balancer_vm_port.rb

```diff
@@ -58,9 +58,14 @@ def health_check_cmd(type)
   end
 
   def check_pulse(session:, previous_pulse:)
-    reading_ipv4, reading_ipv6 = health_check(session:)
-    reading = (reading_ipv4 == "up" && reading_ipv6 == "up") ? "up" : "down"
-    pulse = aggregate_readings(previous_pulse:, reading:, data: {ipv4: reading_ipv4, ipv6: reading_ipv6})
+    pulse = if stack
```

Comment:
The migration leaves us in a temporary state where we have multiple load balancer vm ports for the same load balancer vm and port, one with `NULL` stack and one or two with ipv4 or ipv6 stack.

I think that means `check_pulse` will be duplicating work, at least temporarily. While that may not be a problem, we may want to change callers such that if there is a load balancer vm port with a non-NULL stack, then the load balancer vm port with a NULL stack is ignored.

Reply:
I added the filter to the `./bin/monitor`, this way, once we run the migration and deploy the next commit, we'll perform the health checks only on the `load_blaancer_vm_port`s with `stack` column set to either `ipv4` or `ipv6`.

Reply:
I think this is fine if we deploy the next commit right after running the migration in the previous commit. There may be a short period where it is monitoring both, but it's unlikely to cause any significant issues.